src/python/openvino_genai/py_openvino_genai.pyi

```diff
@@ -0,0 +1,1648 @@
+    def m_generation_ids(self) -> list:
+        ...
+    @m_generation_ids.setter
+    def m_generation_ids(self, arg1: list[str]) -> None:
+        ...
+    @property
+    def m_request_id(self) -> int:
+        ...
+class GenerationStatus:
+    """
+    Members:
+    
+      RUNNING
+    
+      FINISHED
+    
+      IGNORED
+    
+      DROPPED_BY_PIPELINE
+    
+      DROPPED_BY_HANDLE
+    """
+    DROPPED_BY_HANDLE: typing.ClassVar[GenerationStatus]  # value = <GenerationStatus.DROPPED_BY_HANDLE: 4>
+    DROPPED_BY_PIPELINE: typing.ClassVar[GenerationStatus]  # value = <GenerationStatus.DROPPED_BY_PIPELINE: 3>
+    FINISHED: typing.ClassVar[GenerationStatus]  # value = <GenerationStatus.FINISHED: 1>
+    IGNORED: typing.ClassVar[GenerationStatus]  # value = <GenerationStatus.IGNORED: 2>
+    RUNNING: typing.ClassVar[GenerationStatus]  # value = <GenerationStatus.RUNNING: 0>
+    __members__: typing.ClassVar[dict[str, GenerationStatus]]  # value = {'RUNNING': <GenerationStatus.RUNNING: 0>, 'FINISHED': <GenerationStatus.FINISHED: 1>, 'IGNORED': <GenerationStatus.IGNORED: 2>, 'DROPPED_BY_PIPELINE': <GenerationStatus.DROPPED_BY_PIPELINE: 3>, 'DROPPED_BY_HANDLE': <GenerationStatus.DROPPED_BY_HANDLE: 4>}
+    def __eq__(self, other: typing.Any) -> bool:
+        ...
+    def __getstate__(self) -> int:
+        ...
+    def __hash__(self) -> int:
+        ...
+    def __index__(self) -> int:
+        ...
+    def __init__(self, value: int) -> None:
+        ...
+    def __int__(self) -> int:
+        ...
+    def __ne__(self, other: typing.Any) -> bool:
+        ...
+    def __repr__(self) -> str:
+        ...
+    def __setstate__(self, state: int) -> None:
+        ...
+    def __str__(self) -> str:
+        ...
+    @property
+    def name(self) -> str:
+        ...
+    @property
+    def value(self) -> int:
+        ...
+class Generator:
+    """
+    This class is used for storing pseudo-random generator.
+    """
+    def __init__(self) -> None:
+        ...
+class ImageGenerationConfig:
+    """
+    This class is used for storing generation config for image generation pipeline.
+    """
+    adapters: AdapterConfig | None
+    generator: Generator
+    guidance_scale: float
+    height: int
+    negative_prompt: str | None
+    negative_prompt_2: str | None
+    negative_prompt_3: str | None
+    num_images_per_prompt: int
+    num_inference_steps: int
+    prompt_2: str | None
+    prompt_3: str | None
+    strength: float
+    width: int
+    def __init__(self) -> None:
+        ...
+    def update_generation_config(self, **kwargs) -> None:
+        ...
+    def validate(self) -> None:
+        ...
+class LLMPipeline:
+    """
+    This class is used for generation with LLMs
+    """
+    def __call__(self, inputs: openvino._pyopenvino.Tensor | TokenizedInputs | str | list[str], generation_config: GenerationConfig | None = None, streamer: typing.Callable[[str], bool] | StreamerBase | None = None, **kwargs) -> typing.Any:
```

Comment:
not sure why return value is `typing.Any` ? instead of `Union[EncodedResults, DecodedResults]`

Reply:
Looks like `typing.Any` is set everywhere where pybind couldn't derive the type. 
Added the explicit type annotation.